Dfe.Data.SearchPrototype/Web/Program.cs

```diff
@@ -42,6 +43,7 @@
     app.UseHsts();
 }
 
+app.UseDeveloperExceptionPage();
```

Comment:
this should only be used if env.IsDevelopment()

Reply:
Good spot, thanks. Now corrected.